tests/data/ssg-rhel7-ds-tailoring.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xccdf:Tailoring xmlns:xccdf="http://checklists.nist.gov/xccdf/1.2" id="xccdf_scap-workbench_tailoring_default">
```

Comment:
@jameerpathan111 Is this tailoring file safe to upload on github, cause I see `.gov` in link?

Reply:
@jyejare I am not sure. 
What do you think we should do?

Reply:
Well I found this statement on that website "The National Checklist Program (NCP), defined by the NIST SP 800-70, is the U.S. government repository of **publicly available security checklists (or benchmarks)** that provide detailed low level guidance on setting the security configuration of operating systems and applications. "

Reply:
Make Sense! And thanks finding its public availability.